govtool/frontend/src/pages/DRepDirectoryContent.tsx

```diff
@@ -209,12 +209,7 @@ export const DRepDirectoryContent: FC<DRepDirectoryContentProps> = ({
         >
           {dRepList?.length === 0 && <EmptyStateDrepDirectory />}
           {dRepListToDisplay?.map((dRep) => {
-            if (
-              isSameDRep(dRep, myDrep?.view) ||
-              isSameDRep(dRep, inProgressDelegation)
-            ) {
-              return null;
-            }
+            if (dRep.doNotList) return null;
```

Comment:
Actually the problem is somewhere during registering, I've handled filtering here https://github.com/IntersectMBO/govtool/blob/develop/govtool/frontend/src/pages/DRepDirectoryContent.tsx#L102